NPE in PostUploadService > getMediaFileName()

Four crash reports from v2.6.3:

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:300)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
at java.util.concurrent.FutureTask.run(FutureTask.java:242)
at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
at java.lang.Thread.run(Thread.java:841)
Caused by: java.lang.NullPointerException
at org.wordpress.android.util.PostUploadService$UploadPostTask.getMediaFileName(PostUploadService.java:785)
at org.wordpress.android.util.PostUploadService$UploadPostTask.uploadMediaFile(PostUploadService.java:589)
at org.wordpress.android.util.PostUploadService$UploadPostTask.doInBackground(PostUploadService.java:260)
at org.wordpress.android.util.PostUploadService$UploadPostTask.doInBackground(PostUploadService.java:136)
at android.os.AsyncTask$2.call(AsyncTask.java:288)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
... 4 more
```

--- a/src/org/wordpress/android/util/PostUploadService.java
+++ b/src/org/wordpress/android/util/PostUploadService.java
@@ -782,9 +782,9 @@ public class PostUploadService extends Service {
             
             if (!TextUtils.isEmpty(mimeType)) { //try to get the extension from mimeType
                 MimeTypeMap mimeTypeMap = MimeTypeMap.getSingleton();
-                String fileExtensionFromMimeType = mimeTypeMap.getExtensionFromMimeType(mimeType).toLowerCase();
+                String fileExtensionFromMimeType = mimeTypeMap.getExtensionFromMimeType(mimeType);
                 if (!TextUtils.isEmpty(fileExtensionFromMimeType)) {
-                    originalFileName += "." + fileExtensionFromMimeType;
+                    originalFileName += "." + fileExtensionFromMimeType.toLowerCase();
                 } else {
                     //We're still without an extension - split the mime type and retrieve it
                    String[] split = mimeType.split("/");

